documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
It is an option, yeah.  You can put a section in any *.conf file under the service's confdir and the Trapperkeeper app will be able to use it.  Just thought I'd bring that up because we have some flexibility on where it could go, seems like we have a lot already in `puppetserver.conf`, and that there may be some benefit to reducing upgrade pain later if it lives in a different file.